irodsinfo/views_api.py

```diff
@@ -0,0 +1,49 @@
+"""REST API views for the irodsinfo app"""
+
+import logging
+
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+# Projectroles dependency
+from projectroles.plugins import get_backend_api
+
+from irodsinfo.views import IrodsConfigMixin
+
+
+logger = logging.getLogger(__name__)
+
+
+class IrodsConfigRetrieveAPIView(IrodsConfigMixin, APIView):
```

Comment:
One naming convention related thing I missed earlier:

The original UI view returns both the iRODS client environment and a possible certificate file. We refer to the both of these as "iRODS configuration". However, this view only returns the client environment file and nothing else.

Hence, for clarity to the user, we should edit the view, url and docstring to reflect that only the environment is returned.

My suggestion:

- Rename this API view to `IrodsEnvRetrieveAPIView`
- Rename the URL name to `api_env`
- Change the URL pattern to `api/environment`
- Update docstring(s) to state we specifically return the iRODS client environment.

Sorry for not catching this earlier.